doc/release-notes/0.24/0.24.md

```diff
@@ -99,6 +99,12 @@ The following table covers notable changes in v0.24.0. Further information about
 <td valign="top">A new <tt>abortonerror</tt> suboption for <tt>-Xcheck:jni</tt> provides diagnostic data when fatal JNI errors occur. For more information, run <tt>-Xcheck:jni:help</tt>. </td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/11184">#11184</a></td>
+<td valign="top">New Java dump content on processor features</td>
+<td valign="top">All versions</td>
+<td valign="top">The ENVINFO section of a Java dump file now includes further information about processor features to help diagnose problems with AOT compilations that depend on underlying hardware. </td>
```

Comment:
I think this should read "JIT and AOT compilations" because there are two new javacore fields introduced, one for each, and they may differ.

Reply:
Fixed - thanks.